libs/covid/data-api/src/lib/modules/lockdowns/lockdowns.service.ts

```diff
@@ -343,3 +351,21 @@ export class LockdownsService extends BaseService<Lockdown> {
     };
   }
 }
+
+interface ClaimWithLockdowns extends CountyClaim {
+  lockdowns: Lockdown[];
+}
+
+interface LockdownExtended extends Lockdown {
+  claims: ClaimWithLockdowns[];
+  statuses: EntityStatus[];
+}
+
+interface LockdownStat {
+  claims?: number;
+  lockdowns?: number;
+}
+
+export interface LockdownStats {
+  [key: string]: LockdownStats;
+}
```

Comment:
These don't appear to be used anywhere. They have structure to the interfaces in the counties service.

Clean up any unused import statements.

Reply:
deleted it